ksql-engine/src/main/java/io/confluent/ksql/function/udf/math/Ceil.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2018 Confluent Inc.
```

Comment:
nit: 2019

Reply:
so many derps 🤦‍♂ - this is what I get for thinking this was an "easy PR" and not paying attention to details. Thanks for catching them @stevenpyzhang!